Source/views/editProfile.hbs

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
```

Comment:
Niet in lijn met het ontwerp van de rest van het project. De knop is lichtpaars (maak deze blauw).

Het is eventueel mooier om bijde edit functies in 1 tabel te zetten (password en name)

Reply:
De knop is bij deze nog lichtpaars.